components/Badge/badge.css

```diff
@@ -55,10 +55,11 @@
 }
 .badge-icon .badge{
     border-radius: 50%;
-    background-color: var(--clr-high-green);
+    background-color: var(--primary-clr);
     position: absolute;
-    font-size: 1.75rem;
+    font-size: 1.5rem;
     top: 0rem;
```

Comment:
use top 0 and left 0 in HTML tag so you don't have to write it will inherit by its child components. 
And you don't have to use rem when you use 0.

Reply:
Actually then the top:0 will reflect everywhere right, in someplaces I am using top 1rem,2rem, so then anyway I have to write top:1rem or 2rem,right?

Reply:
The child class values override the values of the parent class, i.e when you define globally it will inherit from its parent but if you want to change it you can, by giving different values in child class or id.